gradle/java_connector_python_packaging/build.gradle

```diff
@@ -0,0 +1,252 @@
+}
+
+[BUILD_PYTHON_PACKAGE_TASK_NAME, PUBLISH_PYTHON_PACKAGE_TASK_NAME, PUBLISH_PYTHON_PACKAGE_TO_GCS_TASK_NAME].each { registerGlobalTask(it) }
+
+configure(subprojects.findAll {it.path.startsWith(":airbyte-integrations:connectors:")}) {
+    it.tasks.register(BUILD_PYTHON_PACKAGE_TASK_NAME) {
+        dependsOn = ["assemble"]
+        group = "custom"
+        description = "Builds Python packages for the connector. " +
+                "This will build two packages: one for the connector itself and one for the Java runtime."
+        doLast {
+            def connectorType = project.name.split('-')[0]
+            def connectorName = project.name.split('-')[1..-1].join('-')
+            def connectorPackageName = project.name.replace('-', '_')
+            def connectorProjectName = connectorProjectName()
+            def javaRuntimePackageName = connectorPackageName + "_java"
+            def javaRuntimeProjectName = javaRuntimeProjectName()
+            def javaRuntimeWheelName = "airbyte_" + javaRuntimePackageName
+            def javaRuntimeDirNameInPackage = JAVA_RUNTIME_DIR_NAME
+
+            def valueMappingsForTemplates = [
+                    CONNECTOR_TYPE : connectorType,
+                    CONNECTOR_NAME : connectorName,
+                    CONNECTOR_PACKAGE_NAME : connectorPackageName,
+                    CONNECTOR_PROJECT_NAME : connectorProjectName,
+                    CONNECTOR_VERSION: getConnectorVersion(),
+                    JAVA_RUNTIME_DIR_NAME: javaRuntimeDirNameInPackage,
+                    PLATFORM: detectPlatform(),
+                    JAVA_RUNTIME_PACKAGE_NAME: javaRuntimePackageName,
+                    JAVA_RUNTIME_PROJECT_NAME: javaRuntimeProjectName,
+                    JAVA_RUNTIME_WHEEL_NAME: javaRuntimeWheelName,
+                    JAVA_RUNTIME_GCS_BUCKET_PATH: JAVA_RUNTIME_GCS_BUCKET_PATH
+            ]
+            def connectorPythonProjectDir = file(CONNECTOR_PYTHON_PROJECT_DIR)
+            connectorPythonProjectDir.deleteDir()
+            mkdir(connectorPythonProjectDir)
+
+            def connectorPackageDir = file("${connectorPythonProjectDir}/${connectorPackageName}")
+            if (connectorPackageDir.exists()) connectorPackageDir.deleteDir()
+            mkdir(connectorPackageDir)
+
+            buildPythonPackage(connectorPythonProjectDir, connectorPackageDir, valueMappingsForTemplates)
+
+            def javaRuntimePythonProjectDir = file(JAVA_RUNTIME_PYTHON_PROJECT_DIR)
+            javaRuntimePythonProjectDir.deleteDir()
+            mkdir(javaRuntimePythonProjectDir)
+
+            def javaRuntimePackageDir = file("${javaRuntimePythonProjectDir}/${javaRuntimePackageName}")
+            if (javaRuntimePackageDir.exists()) javaRuntimePackageDir.deleteDir()
+            mkdir(javaRuntimePackageDir)
+
+            buildJavaRuntimePackage(javaRuntimePythonProjectDir, javaRuntimePackageDir, valueMappingsForTemplates, javaRuntimeDirNameInPackage)
```

Comment:
we can get rid of this if we want to since we don't really need to publish a runtime per connector. We can publish one runtime for all connectors. 

When I started working on this this made more sense since I was creating different java runtimes per connector, but turns out it is not straightforward to get all dependencies needed for a specific connector.